inst/rmarkdown/templates/ims/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,414 @@
+---
+title: A sample article with some additional note
+thanks: # delete thanks field if not require, here notes for title
+  - ref: T1 
+    text: A sample additional note to the title.
+runtitle: A sample running head title
+author:
+  - name: First 
+    surname: Author
+    email: first@somewhere.com
+    label: e1
+    addressLabel: A
+    sepNext: "," # use "," or \and; the last author not need this field
+  - name: Second
+    surname: Author
+    email: second@somewhere.com
+    label: e2,mark
+    addressLabel: B
+    sepNext: \and
+  - name: Third
+    surname: Author
+    email: third@somewhere.com
+    label: e3,mark
+    addressLabel: B
+affiliation:
+  - label: A
+    name: Department, University or Company Name
+    authorsLabels: e1
+  - label: B
+    name: Department, University or Company Name
+    authorsLabels: e2,e3
+abstract: |
+  The abstract should summarize the contents of the paper.
+  It should be clear, descriptive, self-explanatory and not longer
+  than 200 words. It should also be suitable for publication in
+  abstracting services. Formulas should be used as sparingly as
+  possible within the abstract. The abstract should not make
+  reference to results, bibliography or formulas in the body
+  of the paper---it should be self-contained.
+
+  This is a sample input file.  Comparing it with the output it
+  generates can show you how to produce a simple document of
+  your own.
+keyword-subclass: | 
+ \begin{keyword}[class=MSC2020] % It must be define for aap, aop, aos journals. For aoas, sts is not used
+ \kwd[Primary ]{00X00}
+ \kwd{00X00}
+ \kwd[; secondary ]{00X00}
+ \end{keyword}
```

Comment:
Is this fixed for the templates ? or should it be modified by the user ? 
I was under the impression of the former and then it should be in the `template.tex` but I may be wrong ?

Reply:
It should be modified by the user.

Reply:
Oh ok. So it made sense to have it commented or not commented.

I saw you followed the idea I suggested to have the function handle the variable. I suggested that thinking it would be removed from the YAML. 😅 

Anyway, if you're happy with the change to provide the `journal` in the function, it is ok with me too.

Reply:
Thanks for the help, I'm happy with the change, I hope it makes it easier for users